packages/nvidia-container-toolkit/nvidia-container-toolkit-config.toml

```diff
@@ -0,0 +1,8 @@
+disable-require = false
+
+[nvidia-container-cli]
+root = "/"
+path = "/usr/bin/nvidia-container-cli"
+environment = []
+debug = "/var/log/nvidia-container-runtime-hook.log"
+ldconfig = "@/sbin/ldconfig"
```

Comment:
This is weird syntax, and it's also strange for `nvidia-container-cli` to be updating the ldconfig cache. Can we either omit this so it doesn't try to `ldconfig`, or give it `/bin/true` instead?